mobile/src/main/res/menu/decklist_menu.xml

```diff
@@ -59,4 +59,20 @@
         android:title="@string/card_view_legality"
         app:showAsAction="ifRoom" />
 
+    <item
+        android:id="@+id/deck_menu_export"
+        android:icon="@drawable/ic_menu_save"
+        android:title="@string/decklist_export"
+        app:showAsAction="ifRoom">
+
+        <menu>
+
+            <item
```

Comment:
This is nested to leave space for other formats?

Reply:
Yep.